main.py

```diff
@@ -110,9 +130,28 @@ def run(self):
 def main():
   socket.setdefaulttimeout(None)
 
+  print 'Creating db connector'
+  db_connector = create_db_connector()
+
+  print 'Fetching current series_for_topic'
+  current_series_for_topics = get_series_for_topics(db_connector)
+
+  print 'Fetching new series_for_topic updates'
+  inserts, updates = generate_series_for_topic_models(current_series_for_topics)
+
+  if inserts:
+    print 'Inserting new series_for_topics'
+    db_connector.write_batch('series_for_topic', inserts)
+
+  print 'Updating series_for_topics'
+  update_series_for_topics(db_connector, updates)
```

Comment:
Do we know if the top series get updated in real time and not on a daily basis?
It would be unnecessar to have this run every 20 minutes or so if it is not getting updated anyways.

Reply:
it is on a daily basis, but this code takes a few seconds to run so I think the convenience of not running another cron job is worth it